src/data/registry.xml

```diff
@@ -498,24 +581,83 @@
               units="fraction" type="real" kind="kind_phys"
               allocatable="allocatable">
       <dimensions>horizontal_dimension</dimensions>
+      <initial_value>0.0_kind_phys</initial_value>
       <ic_file_input_names>landfrac cam_in_landfrac</ic_file_input_names>
     </variable>
+    <variable local_name="icefrac"
+              standard_name="sea_ice_area_fraction"
+              units="fraction" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension</dimensions>
+      <initial_value>0.0_kind_phys</initial_value>
+      <ic_file_input_names>icefrac cam_in_icefrac</ic_file_input_names>
+    </variable>
+    <variable local_name="snowhland"
+              standard_name="lwe_surface_snow_depth_over_land"
+              units="m" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension</dimensions>
+      <initial_value>0.0_kind_phys</initial_value>
+      <ic_file_input_names>snowhland cam_in_snowhland</ic_file_input_names>
+    </variable>
+    <variable local_name="sst"
+              standard_name="sea_surface_temperature"
+              units="K" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension</dimensions>
+      <!-- FIXME: this is a temporary initialization to allow CAM4/CAM7 suites to run standalone,
+           while SSTs are not being provided from another Earth system component. -->
+      <initial_value>273.15_kind_phys</initial_value>
+      <ic_file_input_names>sst cam_in_sst</ic_file_input_names>
+    </variable>
 
-    <!-- Zhang McFarlane (ZM) Variables -->
+    <!-- Temporary initializations from snapshot -->
+    <!-- Note that no_deep_pbl defaults to false, but needs to be set to true for diag_TKE or UW schemes -->
     <variable local_name="pblh"
               standard_name="atmosphere_boundary_layer_thickness"
               units="m" type="real" kind="kind_phys"
               allocatable="allocatable">
       <dimensions>horizontal_dimension</dimensions>
+      <!-- FIXME: this is a temporary initialization to allow CAM4/CAM7 suites to run standalone,
+           while a PBL scheme is unavailable. -->
+      <initial_value>0.0_kind_phys</initial_value>
       <ic_file_input_names>pblh pbuf_pblh</ic_file_input_names>
     </variable>
     <variable local_name="tpert"
               standard_name="convective_temperature_perturbation_due_to_pbl_eddies"
               units="K" type="real" kind="kind_phys"
               allocatable="allocatable">
       <dimensions>horizontal_dimension</dimensions>
+      <initial_value>0.0_kind_phys</initial_value>
       <ic_file_input_names>tpert pbuf_tpert</ic_file_input_names>
     </variable>
+
+    <!-- Cloud fraction variables - need to be kept in physics state -->
```

Comment:
Perhaps a naive comment, but what does "need to be kept in physics state" mean?
1. If it means it should be part of the physics state DDT, then it (and convect_shallow_use_shfrc) should be added to that DDT around line~399. 
2. If it means it should be kept in the registry (in contrast to the temporary variables), then maybe a verbiage change here and below to "need to be kept in the CAM registry"?
3. If it means some third thing... what is it? :)

Reply:
Ah, then I think updating it to refer to the registry rather than the physics state would be best. The physics_state DDT is for quantities that define the "state" of the atmosphere (like temperature, pressure, etc.). 

Also, the generated code from the registry is in `physics_types` rather than physics_state (which is a DDT within physics_types. So maybe what you meant to begin with was "kept in physics types"? If so, that would work for me as well.

Reply:
And I have just reread your question and feel like I didn't quite answer it.

I believe the physics_state DDT primarily exists for the dycore since that expects a state DDT, and then the framework handles the physics interfaces (which don't really care where the data is coming from - DDT or otherwise)

Reply:
Thanks @jimmielin !